src/evaluators/loader.py

```diff
@@ -17,3 +28,30 @@ def load_infmax_model(config: dict[str, Any], random_seed: int, nb_seeds: int) -
         return load_model({"model": config})
     else:
         return lambda x: "dupa"  # TODO: add here GT loader
+
+
+def if_stochastic(infmax_model: Callable) -> bool:
+    match infmax_model.__class__.__name__:
+        case "MultiNode2VecKMeans":
+            return True
+        case "MultiNode2VecKMeansAuto":
+            return True
+    return False
+
+
+def get_seed_sets(
+    infmax_models: dict[str, Callable],
+    net: nd.MultilayerNetworkTorch,
+    repetitions_diffusion: int,
+) -> list[SeedSet]:
+    """Obtain seed sets for a given infmax model on a given network and if needed repeat it."""
+    seed_sets = []
+    for ifm_name, ifm_obj in infmax_models.items():
+        repetitions_infmax=repetitions_diffusion if if_stochastic(ifm_obj) else 1
```

Comment:
[3] Code not formatted

Reply:
I'm using 100 chars line limit and this function goes along the lines of this assumption

Reply:
I've added the whitespaces. W.R.T. chars limit - I'm not sure wether we follow 79 limit in other reopsitories. Nonetheless, that's very minor issue and I'd not bother ourselves with it ;)